### ISSUE TITLE
Deprecate FiberLocal

### ISSUE THREAD

**Issue by jdegoes** [zio/zio#1098]
Now that we have FiberRef, FiberLocal should be deprecated with @deprecated annotations.

**Comment by ywang412** [comment]
I am hoping to make my first contribution to an open source project. Can I work on this issue? Do I need to join the organization? Thank you very much!

**Comment by kaishh** [comment]
@ywang12 Yep, just open a PR. You don't have to join

**Comment by harmeetsingh0013** [comment]
Hey @jdegoes and @kaishh, `@deprecated` annotation accepts two parameters and generally, the second parameter `since` is related with version. Could you please suggest, what will be the value of version?

**Comment by harmeetsingh0013** [comment]
Hey @jdegoes, there is no update on the issue, that's why I started in that. @ywang412 I don't want to steal the issue, but for first-time open source contribution, this looks easy and happy to learn. 

@jdegoes while running `test` on my system, I got an exception: 
```
java.io.IOException: Cannot run program "node": error=2, No such file or directory
[error] 	at java.lang.ProcessBuilder.start(ProcessBuilder.java:1048)
[error] 	at org.scalajs.jsenv.ExternalJSEnv$AbstractExtRunner.startVM(ExternalJSEnv.scala:143)
[error] 	at org.scalajs.jsenv.ExternalJSEnv$AsyncExtRunner.startExternalJSEnv(ExternalJSEnv.scala:222)
```

As per documentation, all the steps perform, but as per exception, some folder is missing. could you please help, how can I resolve this?

**Comment by softinio** [comment]
> there is no update on the issue,

@ywang412 only took this issue less than 24 hours ago

**Comment by harmeetsingh0013** [comment]
@softinio thanks for the update. I am a newbie here, could you please suggest, when will we pick an issue? 
@softinio actually this one really simple one and don't take more than an hour, so I  thought to pick this.

**Comment by softinio** [comment]
I think it would be the right thing to give @ywang412 more time to work on this issue. He is reading the comments on this thread if he is happy for you to take it he will let you know. Open Source projects are worked on by volunteers in their personal time even though something may take one hour its fair to give them time to fit it around their busy lives in my opinion. 

There are plenty of other issues:

https://github.com/zio/zio/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22

Of course if no communication for a long time then its reasonable to get it done but he only took it 20 hours ago.

Just my two cents hope helpful to answer your question. Thanks!

**Comment by harmeetsingh0013** [comment]
Thanks @softinio :)

**Comment by ywang412** [comment]
Hi harmeetsingh0013, 

I will make a pull request momentarily. Thank you very much for stepping in. I volunteered for this issue because it is a low-hanging fruit and its priority is low. I wish to learn from this big open source community. In my work, I tend to let new joiners to pick some easy scrum stories so they feel welcomed and the team will grow. I came to this repository because a coworker highly recommend this repository and we are sure not if to go production with it.    

Thank you!
Yu

**Comment by ywang412** [comment]
I've created a pull request here and went through steps in "ZIO Contributor Guidelines". 
https://github.com/zio/zio/pull/1106

Would zio maintainer kindly review it?

Upon running test cases, I found one test case in SystemSpec is not friendly to Windows native developer. 
  def env1 = {
    val io = unsafeRun(system.env("PATH"))
    io must beSome
    io.get must contain("/bin")
  }

The "/bin" will fail windows platform and "\bin" will work. This test case is platform dependent. I'd like to create an issue for that. 

Thanks!
